src/core/lib/gprpp/bitset.h

```diff
@@ -163,7 +163,7 @@ class BitSet {
                           Int>::type
   ToInt() const {
     Int result = 0;
-    for (size_t i = 0; i < kTotalBits; i++) {
+    for (int i = 0; i < kTotalBits; i++) {
```

Comment:
@ctiller, can you please provide a sanity check that this is safe?

Reply:
eh... it's *probably* safe but weird enough that I'm curious to see the warning/error that triggered the change in case something systemically is causing these odd conversions.

Reply:
changed to 
for (size_t i = 0; i < kTotalBits; i++) {
      if (is_set(static_cast<int>(i))) result |= (Int(1) << i);
    }